implementations/elixir/ockam/ockam/test/ockam/secure_channel_test.exs

```diff
@@ -265,18 +245,255 @@ defmodule Ockam.SecureChannel.Tests do
                    10_000
   end
 
+  test "local secure channel" do
+    {:ok, vault} = SoftwareVault.init()
+    {:ok, alice, alice_id} = Identity.create(@identity_impl)
+
+    {:ok, listener} =
+      SecureChannel.create_listener(
+        identity: alice,
+        encryption_options: [vault: vault]
+      )
+
+    _listener_pid = Ockam.Node.whereis(listener)
```

Comment:
Do we just want to establish this is a living process and not nil? Can this line be dropped?

Reply:
it's completely useless :+1:     (was a leftover,   wanted to monitor the process and check that it terminated when the connecting side terminates,  but that is done on the spawned receiver,  not this process that is the listener itself)